### PR TITLE
adds sdk for pytorchjob from OpenAPI Spec

### PR DESCRIPTION
**Pull request by swiftdiaries**
Similar issue in tf-operator: https://github.com/kubeflow/tf-operator/issues/167
I'll follow-up with another PR to create the actual client sdk.

/cc @johnugeorge @richardsliu

### CODE REVIEW

File: generate-openapi-swagger.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# How to Generate an OpenAPI Spec for PyTorch Operator
+
+From the root of the repo,
+`go run hack/gen-openapi-spec/main.go v0.6 > pkg/apis/pytorch/v1beta2/swagger.json`
```
Comment (richardsliu) [inline generate-openapi-swagger.md]: Same for tfjob - please use v1.

Comment (swiftdiaries) [reply generate-openapi-swagger.md]: Thanks :) 👍 I'll change it

**Comment by swiftdiaries** [comment]
I'm not seeing a diff in the v1beta2 and v1 APISpec for the operator, is this expected?

File: generate-openapi-swagger.md (hunk 1 above)
Comment (swiftdiaries) [reply generate-openapi-swagger.md]: Added it in

**Comment by jinchihe** [comment]
@swiftdiaries Would you mind to close this PR? That has been implemented in the #227 Thanks.

**Pull request closed without merging**